docs/manage/CLI.md

```diff
@@ -347,6 +347,10 @@ viam data export --destination=/home/robot/data --data_type=tabular \
 # export binary data from all orgs and locations, component name myComponent
 viam data export --destination=/home/robot/data --data-type=binary \
 --component-name myComponent
+
+# configure a new database user for use with data query
+viam data database hostname --org-id=abc
+viam data database configure --org-id=abc --password=my_new_secure_password123
```

Comment:
You would configure a database user for the Viam org's Atlas Data Federation instance before grabbing the hostname to that instance, so switch the two here.

If the user has already previously been configured, you can also run the `configure` command to reset the password of an existing database user.

https://github.com/viamrobotics/api/blob/020985a15c113694e4589f725813b9077b6050b0/proto/viam/app/data/v1/data.proto#L61-L67